filament/src/View.cpp

```diff
@@ -335,6 +335,25 @@ void FView::prepare(FEngine& engine, DriverApi& driver, ArenaScope& arena,
         filament::Viewport const& viewport, float4 const& userTime) noexcept {
     JobSystem& js = engine.getJobSystem();
 
+    engine.destroy(mRenderTarget);
+    mRenderTarget = nullptr;
+
+    if (mLDRTexture || mHDRTexture || mDepthStencilTexture) {
+
+        RenderTarget::Builder builder;
+        if (mLDRTexture) {
+            builder.texture(RenderTarget::AttachmentPoint::COLOR0, mLDRTexture);
+        }
+        if (mHDRTexture) {
+            builder.texture(RenderTarget::AttachmentPoint::COLOR1, mHDRTexture);
+        }
+        if (mDepthStencilTexture) {
+            builder.texture(RenderTarget::AttachmentPoint::DEPTH , mDepthStencilTexture);
+        }
+        mRenderTarget = upcast(builder.build(engine));
```

Comment:
Do we have to build this render target? What is it used for besides `viewRenderTarget == mRenderTarget` evaluating to `false` in `Renderer::renderJob()`?